ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -134,6 +152,10 @@ def _map_sample(self, sample, exclude_fields):
             rendered = jinja2.Template(
                 self.template, undefined=jinja2.StrictUndefined
             ).render(**filtered_sample, json_dump=json.dumps, text_proc=text_proc)
+            if rendered.strip() == "":
+                sample[self.new_column_name] = ""
```

Comment:
If you want to reject the sample entirely, maybe just return `None` here to make the subsequent filter trivial (eg `lambda sample: sample is not None`).

Reply:
also, is it worth logging here?

Reply:
Ah I realized I had made a mistake here, and its actually supposed to check the output of textproc(sample.text), not rendered. Rendered is never going to be empty here because there's the other template text. I made a change to pass in the column we want to filter in the args instead.